packages/react/src/forms/FormControl/FormControl.tsx

```diff
@@ -88,7 +91,9 @@ const Root = ({
       {React.Children.map(children, child => {
         if (!React.isValidElement(child)) return
 
-        const describedBy = containsHint ? `${uniqueId}-hint` : undefined
+        const describedBy = [containsHint && `${uniqueId}-hint`, containsValidation && `${uniqueId}-validation`]
```

Comment:
The `aria-describedby` attribute is being set with an empty value when validation isn't needed. Happening because the `undefined` fallback value was removed. Can we add that back in?

Example:

![Screenshot 2024-11-15 at 16 10 04](https://github.com/user-attachments/assets/70246fde-835c-488c-bdf0-0012552c4854)

Reply:
Thanks!